css/styles.css

```diff
@@ -233,6 +233,22 @@ img.animated {
 .attribution p{
   font-size: 1em;
 }
+.page-tear {
+  position: absolute;
+  bottom: 0;
+  background: url("../assets/bg-texture-white-lighter.png") bottom right repeat, url("../assets/section-bottom-tear-black.png") bottom no-repeat;
+  min-height: 45px;
+  width: 103%;
+  margin: -41px;
+  z-index: 2;
+}
+.tear-tan {
+  background: url("../assets/bg-texture-white-lighter.png") bottom right repeat, url("../assets/section-bottom-tear-tan.png") bottom no-repeat;
+}
+.tear-white {
+  /* TODO: add corret texture to white background */
```

Comment:
I don't think the texture works in the white section. I tried the one with no opacity but don't really see it.